base/threading/thread_restrictions.h

```diff
@@ -211,6 +211,11 @@ namespace chrome_cleaner {
 class ResetShortcutsComponent;
 class SystemReportComponent;
 }  // namespace chrome_cleaner
+#if BUILDFLAG(IS_COBALT)
+namespace cobalt {
+class CobaltPathProvider;
```

Comment:
@niranjanyardi Did you look into a starboard-centric way this should be done? Should we create a follow up bug for you to look at potentially changing this or are you fine with this solution? (either way, I think it should be submitted as is)

Reply:
The needed changes here is what I brought up in the chat, and this was the suggested approach. I'm not aware of a followup. This mirrors occurences of `ShellPathProvider` in this file, plus the `#if` to mark our changes to help with rebasing.

Reply:
My understanding of the dependencies and of the chat is that it'd make more sense to me to see USE_STARBOARD here, and define the paths in Starboard.

Reply:
As I said in the chat: This is not something needed for Starboard, but for Cobalt. You can currently build cobalt without Starboard, and changing that here would break that unnecessarily 

Now sometime in the future that, the paths may likely be provided by Starboard, and then the class can be renamed to StarboardPathProvider and this can be changed. However, that is currently not the case.